pom.xml

```diff
@@ -220,6 +220,11 @@
             <artifactId>adventure-text-minimessage</artifactId>
             <version>4.12.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>31.1-jre</version>
+        </dependency>
```

Comment:
This should not be needed, guava 31.1 is already provided by spigot

Reply:
You are correct.